File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.java
class Solution {
    public String solution(String my_string, int s, int e) {
        StringBuilder str = new StringBuilder(my_string);
        
        while(s < e){
            char temp = str.charAt(s);
            str.setCharAt(s, str.charAt(e));
            str.setCharAt(e, temp);
            
            s++;
            e--;
        }
        return str.toString();
    }
}

File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.py
def solution(my_string, s, e):
    my_string = list(my_string)
    
    while s < e:
        my_string[s], my_string[e] = my_string[e], my_string[s]
        
        s += 1
        e -= 1
    return "".join(my_string)